packages/artillery-plugin-publish-metrics/lib/newrelic/index.js

```diff
@@ -113,22 +146,25 @@ class NewRelicReporter {
     return statMetrics;
   }
 
-  // Assembles metrics and info into req body format needed by NR metric API
-  createRequestBody(timestamp, interval, attributeList, metrics) {
+  parseAttributes(attributeList) {
     const parsedAttributes = {};
     if (attributeList.length > 0) {
       for (const item of attributeList) {
-        const attribute = item.split(':');
-        parsedAttributes[attribute[0]] = attribute[1];
+        const [name, ...value] = item.split(':');
+        parsedAttributes[name] = value.join(':');
```

Comment:
Does New Relic allow `:` characters in attribute values?

Reply:
Yes, attributes can be formed with colons `:`. I have tested it and you can see an example on the left side of the screenshot I submitted, under dimensions (attributes show as dimensions in NR UI): one of the attributes submitted is the `target: "http://localhost:3003"`